src/Main.elm

```diff
@@ -376,6 +381,35 @@ view model =
         NotFoundRoute ->
             viewError "Not Found"
 
+---- SUBSCRIPTIONS ----
+
+
+subscriptions : Model -> Sub Msg
+subscriptions model =
+    let
+        comicNumbers =
+            RemoteData.map2 (,)
+                (RemoteData.map .number model.latest)
+                (RemoteData.map .number model.requested)
+
+        handleKeyDown code =
+            case ( code, comicNumbers ) of
+                ( 37, RemoteData.Success ( _, requestedNumber ) ) ->
+                    if requestedNumber /= 1 then
+                        LoadRequestedComic <| requestedNumber - 1
+                    else
+                        Noop
+
+                ( 39, RemoteData.Success ( latestNumber, requestedNumber ) ) ->
+                    if requestedNumber /= latestNumber then
+                        LoadRequestedComic <| requestedNumber + 1
+                    else
+                        Noop
+
+                _ ->
+                    Noop
```

Comment:
`Noop` value isn't really required, IMO.

Reply:
How else would you handle the subscription when the key pressed shouldn't do anything?

Reply:
Oh yes. I totally forgot that. Never mind.